Source/WebDriver/WebDriverService.cpp

```diff
@@ -452,16 +452,14 @@ void WebDriverService::handleMessage(WebSocketMessageHandler::Message&& message,
     }
 
     auto connection = message.connection;
-    auto session = m_bidiServer.session(connection);
-    if (!session) {
-        if (!m_bidiServer.isStaticConnection(connection)) {
-            RELEASE_LOG(WebDriverBiDi, "Unknown connection. Ignoring message.");
-            completionHandler(WebSocketMessageHandler::Message::fail(CommandResult::ErrorCode::InvalidSessionID, connection));
-            return;
-        }
+    RefPtr<Session> session;
+    if (m_bidiServer->sessionID(connection) == m_session->id())
+        session = m_session;
+    else {
+        // FIXME Remove once we support checking static vs non-static methods https://bugs.webkit.org/show_bug.cgi?id=281721
+        completionHandler(WebSocketMessageHandler::Message::fail(CommandResult::ErrorCode::InvalidSessionID, connection));
+        return;
```

Comment:
Why do we need the local variable if we are always returning when not using m_session. Can't we just return and use m_session otherwise?

```
if (m_bidiServer->sessionID(connection) != m_session->id()) {
    // FIXME Remove once we support checking static vs non-static methods https://bugs.webkit.org/show_bug.cgi?id=281721
    completionHandler(WebSocketMessageHandler::Message::fail(CommandResult::ErrorCode::InvalidSessionID, connection));
    return;
}
```